libbeat/docs/command-reference.asciidoc

```diff
@@ -752,7 +752,8 @@ endif::no_dashboards[]
 
 ifdef::has_ml_jobs[]
 * The machine learning jobs contain the configuration information and metadata
-necessary to analyze data for anomalies.
+necessary to analyze data for anomalies. You can only load the jobs to Elasticsearch
+of major version 7. For newer, please use the Machine learning UI in Kibana.
```

Comment:
@dedemorton Likely for later but would be nice to add a link here to the according docs.

Reply:
I don't see a great landing page in the Machine Learning docs. We could send users to the Kibana docs, but then they need to play hopscotch over to the Machine Learning docs. @lcawl Where should this link point?

Reply:
We're in the process of doing a fairly substantial reorganization of the content in the machine learning guide in the main branch. For 7.x, however, if this is referring to the pre-built jobs and you just want to link to reference details about them, it's in https://www.elastic.co/guide/en/machine-learning/7.x/ootb-ml-jobs.html. If you're wanting to link to info about how to add those re-built jobs via Kibana, that's in https://www.elastic.co/guide/en/machine-learning/7.x/create-jobs.html

Reply:
How can I refer to Kibana docs? Just adding the URL as above?